index.html

```diff
@@ -1504,57 +1504,69 @@
-individuals have a collective interest in the creation of information about the group, and actions
-taken on its behalf.</em>” ([[?Individual-Group-Privacy]]) This justifies ensuring that grouped
-people can benefit from both individual and collective means to support their [=autonomy=] with respect
-to [=data=] [=processing=]. It should be noted that [=processing=] can be unjust even if individuals
-remain anonymous, not from the violation of individual [=autonomy=] but because it violates ideals
-of social equality ([[?Relational-Governance]]).
-
-Another case in which collective decision-making is preferable is for [=processing=] for which
-informed individual decision-making is unrealistic (due to the complexity of the processing, the
-volume or frequency of processing, or both). Expecting laypeople (or even experts) to make informed
-decisions relating to complex [=data=] [=processing=] or to make decisions on a very frequent
-basis — even if the [=processing=] is relatively simple — is unrealistic if we also want them to have
-reasonable levels of [=autonomy=] in making these decisions.
-
-The purpose of this principle is to require that [=data governance=] provide ways to distinguish
-[=appropriate=] [=data=] [=processing=] without relying on individual decisions whenever the latter
-are impossible, which is often ([[?Relational-Governance]], [[?Relational-Turn]]).
-
-Which forms of collective governance are recognised as legitimate will depend on domains. These may
-take many forms, such as governmental bodies at various administrative levels, standards
+Collective decision-making is likely preferable:
+
+<ul>
+  <li>
+    When information is about membership in a group or about a group's behaviour.
+    As Brent Mittelstadt explains, “<em>Algorithmically grouped
+    individuals have a collective interest in the creation of information about the group, and actions
+    taken on its behalf.</em>” ([[?Individual-Group-Privacy]])
+    As <a href="#group-privacy"></a> discusses,
+    an individual's permission isn't enough to support the [=autonomy=] of other members of the group.
+  </li>
+
+  <li>
+    When individuals can't realistically be expected to make informed decisions.
+    This can happen when [=data processing=] is complex
+    or requests to process data happen very frequently.
+  </li>
+
+  <li>
+    When individuals have systematically less power
+    than the organizations asking them to agree to [=data processing=] ([[?Relational-Turn]]).
+  </li>
+
+  <li>
+    When the [=data processing=] is justified or unjustified at a societal level,
+    even accounting for individual disagreements ([[?Relational-Governance]]).
+  </li>
+</ul>
+
+Different forms of collective decision-making are legitimate depending on what data is being processed.
+These forms might be governmental bodies at various administrative levels, standards
 organisations, worker bargaining units, or civil society fora.
```

Comment:
Not vital, but wondering if we should reference: https://www.mnot.net/blog/2023/11/01/regulators

Reply:
In a separate PR. :)